### DISCUSSION TITLE
Any plans for typescript / type information in V5?

### DISCUSSION THREAD

**Discussion by mjarkk** [twbs/bootstrap#31587] [General] [answered]
I'm very excited for version 5, this allows the company i work at to remove jquery from the dependencies of some of our products to safe some lovely bytes.  
Since the first v5 alpha i've been experimenting with running our products without jquery and so far it's looking great!!  

Although i have one issue from the programming perspective is that i don't get hints and because of that i constantly need to switch between the bootstrap docs and the code to change code over from jquery to native Typescript.  
Our product is very large and thus lots of code to change, this sadly also means lots of places where bugs can occur. Everything till now seems like it's going good but i wouldn't be surprised if i forget to type 1 letter and some parts of the application won't work anymore, these kinds of bugs could be easily resolved when there where some sort of code definitions.

**Comment by Johann-S** [comment, marked as answer]
Hi @mjarkk,

Currently there is no plan to provide a TypeScript type definition for Bootstrap v5, but I saw a few weeks an update will come on @types/bootstrap package

**Reply by mjarkk** [reply]
Awesome i didn't know that package existed :), thanks
